frontend/public/config.yml

```
@@ -383,6 +383,35 @@ collections:
+      - label: 'Current LARP event'
+        name: 'current_larp_event'
+        file: 'frontend/src/content/pages/current-larp-event.md'
+        fields:
+          - { label: 'Najnovšia udalosť', name: 'current_larp_event', widget: 'relation', collection: 'events', searchFields: ['title'], valueField: 'title', displayFields: ['title'] }
+          - { label: 'Veľký text nad zápletkami', name: 'storyIntroText', widget: 'string', default: 'Na čo sa môžeš tešiť?' }
+          - { label: 'Obrázok pri zápletkách', name: 'storyImage', widget: 'image' }
+          - { label: 'Id eventu v tootoot.fm', name: 'tootoot_id', widget: 'string', hint: 'Napríklad: 65bd0d301b234e08804169cb, nájdeš ho v url konkrétneho eventu na tootoot stránke. Netreba login práva.' }
+          - { label: 'Nadpis nad lístkami', name: 'ticketsTitle', widget: 'string', default: 'Lístky' }
+          - { label: 'Text nad lístkami', name: 'ticketsText', widget: 'markdown' }
+          - {
+              label: 'Popis archivovanej akcie',
```

yamllint (1.35.1)

[error] wrong indentation: expected 12 but found 14

(indentation)